resources/MSBuildImports/15.0/Microsoft.Common.targets/ImportAfter/Microsoft.NuGet.ImportAfter.targets

```diff
@@ -20,4 +20,10 @@ Copyright (c) .NET Foundation. All rights reserved.
     <NuGetRestoreTargets Condition="'$(NuGetRestoreTargets)'==''">$(MSBuildExtensionsPath)\NuGet.targets</NuGetRestoreTargets>
   </PropertyGroup>
   <Import Condition="Exists('$(NuGetRestoreTargets)')" Project="$(NuGetRestoreTargets)" />
+  
+  <!-- Import Pack.targets for pack -->
+  <PropertyGroup>
+    <NuGetRestoreTargets Condition="'$(NuGetRestoreTargets)'==''">$(MSBuildExtensionsPath)\Pack.targets</NuGetRestoreTargets>
```

Comment:
I don't think the `Pack` targets file should be included like this.  Instead, I think it should get imported into the app through a NuGet package, just like the rest of the `Sdk` targets.

In my mind, the only thing that should get installed with the CLI is:
- An MSBuild implementation
- A NuGet "restore" implementation

Everything else - "Build", "Publish", "Pack", "Test" MSBuild targets - can come through a NuGet package.

This allows:

1. It ensures ALL tools (VS, CLI, VS Code, MSBuild command line, etc) use the same build logic.
2. It allows us to update the build logic, add features, fix bugs, etc without shipping a new CLI.